content/stub.xhtml

```diff
@@ -251,15 +251,10 @@
        * message has been focused in Gecko 1.9.1 (so it's basically
        * useless). */
       let msgNodes = document.getElementsByClassName("message");
-      let index;
-      for each (let [i, k] in Iterator(msgNodes)) {
-        if (k == msgNode) {
-          index = i;
-          break;
-        }
+      let index = Array.prototype.indexOf.call(msgNodes, msgNode);
+      if (index != -1) {
+        msgNode.setAttribute("tabindex", index + 2);
       }
-      if (index)
-        msgNode.setAttribute("tabindex", index+2);
```

Comment:
Funny. Because of the way the previous test was phrased, you're actually changing the behavior in case i == 0. But, amusingly enough, this is functionnaly equivalent, because if the first node was set to be focused, then whether it has tabindex 1 or 2 doesn't matter... I think. Anyhow, that seems more than legit :)